### DISCUSSION TITLE
Can't login/register after installation

### DISCUSSION THREAD

**Discussion by juliend2** [Budibase/budibase#10314] [Help] [answered]
I just installed budibase by following the steps over here: https://docs.budibase.com/docs/docker-compose
But after, I go to http://localhost:10000/ and it redirects me to the login screen (http://localhost:10000/builder/auth/login).
I can't find the place where I can register my first user.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @juliend2,

I just tried this going through the whole flow and it hasn't cause any issues for myself? 

To add more users you need to login with the admin account created after running through the whole process. At the end of the video you can see it directs you to a page where you can setup your admin account. Did you perhaps do this and have forgotten the details to the account which you've setup?

I think it would be best to wipe your container and items related to it and run through the setup process again.

I've provided a list of steps below to achieve this:

1. Stop the container `budi hosting --stop` wait for this to stop.
2. Go into your docker application and delete the container (Screenshot attached showing were to do this.)<img width="977" alt="Screenshot 2023-04-17 at 11 56 27" src="https://user-images.githubusercontent.com/126772285/232464917-aeab406f-f6f4-4b69-9250-bfaf59a65c21.png">
3. Prune your couchdb, redis and minio data e.g. `Docker volume rm budibase_redis_data - Docker volume rm budibase_minio_data`. (You can do a `docker volume ls` to find the names of the volumes.)
4. Now do a full system docker prune `docker system prune`
5. Follow the docker compose setup docs again, or try out this for a more in-depth breakdown: https://vimeo.com/showcase/budibase-get-started 

I hope this helps!
